proxy/http/HttpSM.cc

```diff
@@ -2976,7 +2976,10 @@ HttpSM::tunnel_handler_server(int event, HttpTunnelProducer *p)
 {
   STATE_ENTER(&HttpSM::tunnel_handler_server, event);
 
-  milestones[TS_MILESTONE_SERVER_CLOSE] = Thread::get_hrtime();
+  // The following fixes up an intercept handler which doesn't set TS_MILESTONE_SERVER_CONNECT by default.
```

Comment:
To clarify that this comment applies to the check itself rather than the whole conditional block, maybe reword as:

```
  // An intercept handler may not set TS_MILESTONE_SERVER_CONNECT by default. Therefore we only 
  // set TS_MILESTONE_SERVER_CLOSE if TS_MILESTONE_SERVER_CONNECT is set (non-zero), lest 
  // certain time statistics are calculated from epoch time.
```

Just a suggestion.

Reply:
updated PR with above comment.